lldb/cmake/modules/AddLLDB.cmake

```diff
@@ -93,13 +93,6 @@ function(add_lldb_library name)
     set(libkind STATIC)
   endif()
 
-  #PIC not needed on Win
-  # FIXME: Setting CMAKE_CXX_FLAGS here is a no-op, use target_compile_options
```

Comment:
Do we in fact use target_compile_options? Or do we leave PIC up to something else.

Reply:
I believe we don't -- and I think that's the right thing to do  (we should leave it to something else to decide)